### ISSUE TITLE
TypeError: args.includes is not a function

### ISSUE THREAD

**Issue by ludden007** [keystonejs/generator-keystone#243]
Hi... I followed the install instructions for generator-keystone (v0.5.0), and when trying to create a project, received a "TypeError: args.includes is not a function".  I downgraded geneator-keystone to v0.4.2 and it's now working, so something in 0.5.0 is causing this.  Cheers.

```
$ yo keystone
/usr/local/lib/node_modules/generator-keystone/app/index.js:14
	this.auto = args.includes('auto');
	                 ^

TypeError: args.includes is not a function
    at new KeystoneGenerator (/usr/local/lib/node_modules/generator-keystone/app/index.js:14:19)
    at Environment.instantiate (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:336:10)
    at Environment.create (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:313:15)
    at Environment.run (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/environment.js:372:24)
    at /usr/local/lib/node_modules/yo/lib/cli.js:140:11
    at Array.forEach (native)
    at /usr/local/lib/node_modules/yo/lib/cli.js:139:9
    at Environment.resolver.lookup (/usr/local/lib/node_modules/yo/node_modules/yeoman-environment/lib/resolver.js:50:12)
    at init (/usr/local/lib/node_modules/yo/lib/cli.js:115:7)
    at pre (/usr/local/lib/node_modules/yo/lib/cli.js:73:3)
```

```
$ yo doctor

Yeoman Doctor
Running sanity checks on your system

✔ Global configuration file is valid
✔ NODE_PATH matches the npm root
✔ Node.js version
✔ No .bowerrc file in home directory
✔ No .yo-rc.json file in home directory
✔ npm version
```

**Comment by darwinFailure** [comment]
same here

**Comment by Noviny** [comment]
I did some digging in to this, and it appears to be with older versions of node. I am not experiencing the problem on `node 6.3.1`. It wasn't our intent to stop supporting older versions of node, so I'll get a fix out for this.

**Comment by JedWatson** [comment]
Closed by #244
